Recrutify/Recrutify.DataAccess/Repositories/CandidateRepository.cs

```diff
@@ -25,7 +25,7 @@ public IQueryable<Candidate> GetByProject(Guid projectId)
 
         public IQueryable<Candidate> GetCandidatesPassedTestByProject(Guid projectId)
         {
-            return GetCollection().AsQueryable().Where(x => x.ProjectResults.Any(p => p.ProjectId == projectId && p.Status == Status.Test));
+            return GetCollection().AsQueryable().Where(x => x.ProjectResults.Any(p => p.ProjectId == projectId && p.Status == Status.Test && p.Feedbacks.Any(x => x.Rating != 0)));
```

Comment:
You check what any of feedbacks should be not 0. Is it correct?? You should check that test feedback not 0

Reply:
done